schema/cassandra/chardonnay/schema.cql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE range_map (
     key_lower_bound_inclusive    blob,
     key_upper_bound_exclusive    blob,
     assignee                     text,
-    PRIMARY KEY  ((keyspace_id), range_id)
+    PRIMARY KEY  (keyspace_id, range_id)
```

Comment:
I think using the keyspace_id as the partition key is useful, since clients are likely to want to retrieve all the range assignments for a keyspace

Reply:
Done. I thought it was just a typo :)